devices.js

```diff
@@ -4589,13 +4625,18 @@ const devices = [
         vendor: 'SmartThings',
         description: 'Multi Sensor (2015 model)',
         supports: 'contact and temperature',
-        fromZigbee: [fz.temperature, fz.smartsense_multi, fz.ias_contact_alarm_1],
+        fromZigbee: [fz.temperature, fz.smartsense_multi, fz.ias_contact_alarm_1, fz.battery_cr2450],
         toZigbee: [],
         meta: {configureKey: 1},
         configure: async (device, coordinatorEndpoint) => {
             const endpoint = device.getEndpoint(1);
             await bind(endpoint, coordinatorEndpoint, ['msTemperatureMeasurement']);
+            const readPowerConfigClusers = [
+                'batteryVoltage',
+            ];
+            await endpoint.read('genPowerCfg', readPowerConfigClusers);
             await configureReporting.temperature(endpoint);
+            await configureReporting.batteryVoltage(endpoint);
```

Comment:
Please try changing:
-  `fz.battery_cr2450` to `fz.battery` 
- `await configureReporting.batteryVoltage(endpoint);` to `await configureReporting.batteryPercentageRemaining(endpoint);`
- `const readPowerConfigClusers = [
                'batteryVoltage',
            ];` to `const readPowerConfigClusers = [
                'batteryPercentageRemaining',
            ];`

Reply:
I'll need to check this later tonight - but I don't think this sensor gives %age remaining - only battery voltage (it's an old sensor from 2015)

Reply:
Yeah, it only appears to give batteryVoltage:
```
zigbee2mqtt:error 2020-04-28 23:13:03: Failed to configure '0x000d6f000b19539b', attempt 1 (Error: Read 0x000d6f000b19539b/1 genPowerCfg(["batteryPercentageRemaining"], {"timeout":6000,"manufacturerCode":null,"disableDefaultResponse":true}) failed (Error: Status 'UNSUPPORTED_ATTRIBUTE')

```

Also confirmed by popping a new battery in it

Reply:
Thanks for trying!